drafts/GoL-reules/Game of Links.md

```diff
@@ -0,0 +1,139 @@
+# The Game of Links. Goals, rules and provision.
+
+## Intro
+
+
+## General provision of the Game.
+
+![](euler-5_timeline.png)
+
+We want to reproduce the whole process of mainnet launch in `Euler-5` for testing auction, drop and distribution models. As shown in figure above the Game isn't start with `Euler-5` launch. We want to allow some time to participants of the Game for their funds checking and hardware and software setuping. Also, in that 15-days limb the Gamers can decide on what exactly discipline they want to participate and strategize.
+
+`Euler-5` Genesis file will contain all drops to Ethereum and Cosmos communities. Furthermore,  it will contain validators lifetime rewards from `Euler-4`. For new incomers, we'll propose to get CYB tokens at the Auction just for less. In the Auction, newbies can buy CYBs or GOLs for uAtoms and Szabos. 
+
+> ***Few words about the Auction.***
+
+With the Game of Links start after 15 days of `Euler-5` launch, we will start tracking the progress at the Game monitor and store data. Then, and only then. The monitor will be available at the Brain block explorer by cyberd.ai address. 
+
+The end of the Game will define one of 3 control events happens as a result:
+
+-  90 days will be passed after the Game start
+-  500.000 ATOMs well be donated after funding start
+-  Knowledge Graph well be increased by 1B links
+
+In addition to each requirement we add one more: set of `146` active validators must have lasted at least 10.000 blocks. In another case, the Game would be considered as unsuccessful and restarted.
```

Comment:
We discissed with @litvintech that we could set a hard requirement to 73 and assign bonus for reaching 146